mandatory/2-logic-error.js

```diff
@@ -1,16 +1,15 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
 function trimWord(word) {
-  return wordtrim();
+  return trimWord();
 }
 
 function getStringLength(word) {
-  return "word".length();
+  return word.length();
```

Comment:
here length is a property not a method, that is it should not have () parenthesis after the word.